uwblueprint/feeding-canadian-kids
# Update food_request_group and food_request model

## Notion Ticket Link

[Notion Ticket](https://www.notion.so/uwblueprintexecs/Dev-e3112e78136f49b4b042e9a0d9df9723?p=68564c82fa3049a291d2e7e389753f62&pm=s)



## Implementation Description
* Update the food_request_group and food_request models according to the specified schemas in Notion


## Steps To Test
1. Run the newly created test to verify that a food request group can be created with no errors.
2. Verify in MongoDB that the food request group was created and matches the specified schema.



## What Should Reviewers Focus On?
* We would like to make sure that our implementation of the dietaryRestrictions field as a JSONString is suitable, please look at how it is specified in the mutation in the test (note that a \\\\ in python represents a \ in the actual mutation)


## Checklist
